doc/howto/grafana.md

```diff
@@ -5,8 +5,8 @@ relatedlinks: https://grafana.com/grafana/dashboards/19131-lxd/
 (grafana)=
 # Set up a Grafana dashboard
 
-To visualize the metrics and logs data, set up [Grafana](https://grafana.com/).
-LXD provides a [Grafana dashboard](https://grafana.com/grafana/dashboards/19131-lxd/) that is configured to display the LXD metrics scraped by Prometheus and events sent to Loki.
+To visualize the metrics data, set up [Grafana](https://grafana.com/).
```

Comment:
Did you change this back on purpose, or is it a result of merging?
We did some improvements to this file that Incus doesn't have.

Reply:
It's the result of a merge resolution conflict so it was me unintentionally undoing the improvement. I'll tweak those, thanks for catching this!

Reply:
Thanks!